update README to reflect integration of 1091

Martin521 opened the pull request.

bbatsov commented:
I think it'd be prudent to add the missing column everywhere, as noted by `markdown-lint`. Perhaps we can use checkboxes for it instead of plain text?

bbatsov commented:
One more thing - when you mention PRs in commit messages it's best to prefix them with `#` (e.g. #1091), so the auto-linking in GitHub works.

Martin521 commented:
I added the missing columns. Checkboxes won't work because I expect some of the lines to receive entries such as "not implemented" or "already integrated in the original". The reference in the header is not a PR (but thanks for the remark, I have updated it).

edgarfgp reviewed (approved):
Thanks.
